src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -15,18 +16,25 @@
 
 (trapperkeeper/defservice jruby-puppet-pooled-service
                           jruby/JRubyPuppetService
-                          [[:ConfigService get-config]
-                           [:ShutdownService shutdown-on-error]
-                           [:PuppetProfilerService get-profiler]
-                           [:PoolManagerService create-pool]]
+                          {:required [[:ConfigService get-config]
+                                      [:ShutdownService shutdown-on-error]
+                                      [:PuppetProfilerService get-profiler]
+                                      [:PoolManagerService create-pool]]
+                           :optional [MetricsService]}
```

Comment:
Yeah, I think that can simplify the code, and probably makes sense now that we have permission to move all the rest of the metrics to open source. The only thing is that we'll need to add a metrics service to the tk bootstrap, but that shouldn't really be a problem.

My only question is should we have a setting about whether to use http metrics or not? We haven't done any performance testing with the http metrics, and there might be a possibility it could affect performance. I guess I'm thinking it wouldn't be a bad thing to have a way to turn it off.